src/Native/Unix/System.Security.Cryptography.Native/openssl.cpp

```diff
@@ -1242,6 +1271,26 @@ extern "C" int32_t CryptoNative_LookupFriendlyNameByOid(const char* oidValue, co
     return 0;
 }
 
+#ifndef OPENSSL_VERSION
+#define OPENSSL_VERSION 0
+#endif
+
+/*
+Function:
+SSLEayVersion
+
+Gets the version of openssl library.
+
+Return values:
+Textual description of the version on success.
+"not available" string on failure.
+*/
+extern "C" char* CryptoNative_SSLEayVersion()
+{
+    return strdup(OpenSSL_version(OPENSSL_VERSION));
```

Comment:
This should be free'd by the caller, right?

Reply:
Yeah, it's one of the sometimes-nice-sometimes-frustrating things that the Marshaller does.